CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The following is a curated list of changes in the Enact agate module, newest cha
 - `agate/ArcSlider` prop `aria-valuetext` to override `aria-valuetext` for it
 - `agate/Button` prop `iconOnly`, `iconPosition` and `minWidth`
 - `agate/DateTimePicker` prop `dayAriaLabel`, `hourAriaLabel`, `meridiemAriaLabel`, `minuteAriaLabel`, `monthAriaLabel`, and `yearAriaLabel`
+- `agate/Heading` `marquee` to text
```

Comment:
```suggestion
- `agate/Heading` prop `marqueOn`  to select marquee condition
```

@MikyungKim @0x64 
How do you think?

Reply:
### Added

- `agate/Heading` prop `marqueOn` to determine when marquee begins

### Changed

- `agate/Heading` to support marquee